src/main/java/com/kosherjava/zmanim/hebrewcalendar/YerushalmiYomiCalculator.java

```diff
@@ -61,11 +62,11 @@ public static Daf getDafYomiYerushalmi(JewishCalendar calendar) {
 		Calendar requested = calendar.getGregorianCalendar();
 		int masechta = 0;
 		Daf dafYomi = null;
-		
-		// There isn't Daf Yomi in Yom Kippur and Tisha Beav.
+
+		// There isn't Daf Yomi on Yom Kippur or Tisha B'Av.
 		if ( calendar.getYomTovIndex() == JewishCalendar.YOM_KIPPUR ||
-			 calendar.getYomTovIndex() == JewishCalendar.TISHA_BEAV ) {
-			return new Daf(39,0);
+				calendar.getYomTovIndex() == JewishCalendar.TISHA_BEAV ) {
+			return null;
```

Comment:
@Elyahu41 it would seem that this `return null` has caused a regression in com.kosherjava.zmanim.hebrewcalendar.UT_YerushalmiTest#testCorrectSpecialDate 

Please have a look there and fix the test.

Reply:
Thank you @elijulian for catching this. @Elyahu41 introduced the fix for a reason, it caused 2 issues that I did not realize when merging this (and shows the values of running tests, that clearly have to be expanded).

1. The tests no longer pass. I addressed this by changing the test to use asserNull, but...
2. The Daf being null will be that the formatting class has to handle this to return the "No Daf Today" that the formatter returns when there is no Daf.  This was corrected by tweaks to the Daf and formatter classes.